packages/core/database/lib/connection.js

```diff
@@ -17,39 +15,67 @@ const trySqlitePackage = (packageName) => {
     throw error;
   }
 };
+
+/**
+ * SQLite
+ */
+const SqliteClient = require('knex/lib/dialects/sqlite3/index');
+
 class LegacySqliteClient extends SqliteClient {
   _driver() {
     return require('sqlite3');
   }
 }
 
-const clientMap = {
-  'better-sqlite3': 'better-sqlite3',
-  '@vscode/sqlite3': 'sqlite',
-  sqlite3: LegacySqliteClient,
-};
-
 const getSqlitePackageName = () => {
-  // NOTE: allow forcing the package to use (mostly used for testing purposes)
+  // allow forcing the package to use (mostly used for testing purposes)
   if (typeof process.env.SQLITE_PKG !== 'undefined') {
     return process.env.SQLITE_PKG;
   }
 
-  // NOTE: this tries to find the best sqlite module possible to use
-  // while keeping retro compatibility
+  // find the package based on availability
   return (
-    trySqlitePackage('better-sqlite3') ||
-    trySqlitePackage('@vscode/sqlite3') ||
-    trySqlitePackage('sqlite3')
+    tryResolvePackage('better-sqlite3') ||
+    tryResolvePackage('@vscode/sqlite3') ||
+    tryResolvePackage('sqlite3')
   );
 };
 
+/**
+ * MySQL
+ */
+const getMysqlPackageName = () => {
+  // allow forcing the package to use (mostly used for testing purposes)
+  if (typeof process.env.STRAPI_MYSQL_PKG !== 'undefined') {
```

Comment:
what's going on here ? why are you checking for `STRAPI_MYSQL_PKG` and then returning `MYSQL_PKG` shouldn't it be the same ?